src/pyramid_debugtoolbar/panels/templates/request_vars.dbtmako

```diff
@@ -84,7 +84,11 @@
 			</tbody>
 		</table>
 	% else:
-		<p>No POST data</p>
+		<p>No POST variables</p>
+		% if post_raw:
+			<h4>POST Raw Data</h4>
```

Comment:
Maybe note something like `(first 4096 bytes)` or add some `... <N more bytes>` thing at the end of `post_raw`.

Also I'm pretty sure this is broken on python 3, you need to come up with a unicode representation of the byte string before trying to pass it off to mako. The easiest way to do this imo is determine if the request contains utf-8 data and if it doesn't then just show something like `< ... N bytes ...>` instead.

Reply:
you're right it is. good catch.